.github/workflows/build-iso.yml

```
@@ -21,79 +18,85 @@
-          Checksum file is attached to this release.
-
-          ISO Filename: ${{ steps.build.outputs.iso_name }}
-        draft: false
-        prerelease: false
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v4
+
+      - name: Get Release Version
+        id: get_version
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:44: Double quote to prevent globbing and word splitting

(shellcheck)